000-vip-init.php

```diff
@@ -4,6 +4,10 @@
 	require __DIR__ . '/.secrets/vip-secrets.php';
 }
 
+if ( ! defined( 'A8C_PROXIED_REQUEST' ) ) {
+	define( 'A8C_PROXIED_REQUEST', false );
```

Comment:
I'm worried that it's technically possible to define this constant in the user space.

What prevents a user from adding `define( 'A8C_PROXIED_REQUEST', true )` in their `vip-config.php`?